meta-iot2000-example/recipes-example/iot2000setup/iot2000setup_1.0.bb

```diff
@@ -1,23 +1,29 @@
 SUMMARY = "Copy iot2000setup files to file system"
 LICENSE = "MIT"
 LIC_FILES_CHKSUM = "file://${IOT2000_MIT_LICENSE};md5=838c366f69b72c5df05c96dff79b35f2"
+inherit update-rc.d 
 
 SRC_URI = "file://expandfs.sh file://iot2000setup.py"
 
 FILES_${PN} += " \
 	${sysconfdir}/iot2000setup/expandfs.sh \
 	${sysconfdir}/iot2000setup/iot2000setup.py \
+	${sysconfdir}/init.d/expandfs.sh \
 	${bindir}/iot2000setup \
 "
 
 RDEPENDS_${PN} += " libnewt-python"
+INITSCRIPT_NAME = "expandfs.sh"
+INITSCRIPT_PARAMS = "defaults 10"
 
 do_install() {
    install -d ${D}${sysconfdir}/iot2000setup
+   install -d ${D}${sysconfdir}/init.d
    install -d ${D}${bindir}
    install -m 0755 ${WORKDIR}/expandfs.sh ${D}${sysconfdir}/iot2000setup/expandfs.sh
    install -m 0755 ${WORKDIR}/iot2000setup.py ${D}${sysconfdir}/iot2000setup/iot2000setup.py
    ln -sf /etc/iot2000setup/iot2000setup.py ${D}${bindir}/iot2000setup
+   install -m 0755 ${WORKDIR}/expandfs.sh ${D}${sysconfdir}/init.d/expandfs.sh
```

Comment:
Why are you installing `expandfs.sh` a second time, this time to `/etc/init.d/`? Isn't once and then, e.g., symlinking it sufficient? This would give a hint that it's indeed one file which is just placed in `/etc/init.d` for startup via `init`.